config/commands/_search_class.py

```diff
@@ -0,0 +1,55 @@
+#!/Users/bytedance/miniconda3/envs/myenv/bin/python3
+
+# @yaml
+# signature: _search_class <class_name> <project_path>
```

Comment:
I think we may not need <project_path>. because I think the working dir is the <project_path>